skins/pupperdao.scss

```
@@ -0,0 +1,10 @@
+.pupper {
```

The css selector should similar to the skin file name. `skin.scss` `.skin {}`.
```suggestion
.pupperdao {
```